src/pages/simulator.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, Text } from "@chakra-ui/react"
+import React from "react"
+import { walletOnboardingSimData } from "../data/WalletSimulatorData"
+import { Simulator } from "../components/Simulator"
+
+const SimulatorPage = ({ location }) => (
+  <Box w="full" py={16}>
+    <Simulator location={location} data={walletOnboardingSimData}>
+      <Text
+        fontSize={{ base: "lg", md: "xl", lg: "2xl" }}
+        fontStyle="italic"
+        color="body.medium"
+        mb={2}
+      >
+        Interactive explainer
+      </Text>
+      <Text
+        fontSize={{ base: "3xl", md: "4xl", lg: "5xl" }}
+        lineHeight="115%"
+        fontWeight="bold"
+        m={0}
+      >
+        How to use a wallet
```

Comment:
Note: we have merged the new Headings from the DS. So, you could use `<Heading as="h1" size="2xl">` for example. You'd need to merge `dev` first.

Reply:
I think originally I wasn't viewing this as an article/document, so I avoided the `h` tags, but think it's probably better to leave them in. I'll use an `h2` for "How to use a wallet" (since it'll already be nested inside a page with an `h1`... in the modal I'll then use `h3`s for the explanation header text.

Reply:
Cool. In that case you wouldn't need to specify the `as` prop since the default is already `h2`.